python/foglamp/common/configuration_manager.py

```diff
@@ -1026,7 +1015,16 @@ def diff(lst1, lst2):
                 return result
 
         try:
+            deleted_ones = list()
             result = await delete_recursively(category_name)
+            # All children and relationships removed. Remove remaining entries for this cat as child.
+            payload = PayloadBuilder().WHERE(["child", "=", category_name]).payload()
+            result = await self._storage.delete_from_tbl("category_children", payload)
+            if result['response'] == 'deleted':
+                _logger.warning('Deleted parent and/or child entries in catgory_children for %s', category_name)
```

Comment:
its an INFO

Reply:
Better message will be: `Deleted parent child relationship entries for %s', category_name

Reply:
The default log level set is warning and it is mandatory to write to syslog for such events, hence level kept as warning.